crates/astria-core/src/protocol/transaction/v1alpha1/mod.rs

```diff
@@ -282,6 +327,10 @@ enum UnsignedTransactionErrorKind {
     Action(#[source] action::ActionError),
     #[error("`params` field is unset")]
     UnsetParams(),
+    #[error("invalid type URL")]
+    InvalidTypeUrl,
+    #[error("invalid bytes in Any")]
```

Comment:
I would be a tad more specific about this. It might be that 
```suggestion
#[error("failed to decode `google.protobuf.Any` to `{}`", UnsignedTransaction::type_url())]
```